app/pages/resources/contact.tsx

```diff
@@ -21,19 +24,32 @@ class Contact extends Component<Props> {
     }
   `;
 
-  // TODO: Add content to this empty page
   render() {
     const {query} = this.props;
     const {session} = query || {};
+    const supportEmail = getConfig()?.publicRuntimeConfig.SUPPORT_EMAIL;
+    const supportMailToUrl = supportEmail
+      ? `mailto:${supportEmail}?subject=Support Request (CIIP)`
+      : '#';
+    const adminEmail = getConfig()?.publicRuntimeConfig.ADMIN_EMAIL;
+    const adminMailToUrl = adminEmail
+      ? `mailto:${adminEmail}?subject=Support Request (CIIP)`
```

Comment:
Since this is a general contact page and not specific to support requests, a subject line like "CIIP Inquiry" might be more appropriate for the `adminMailToUrl` here.

Conversely, we only ask them to email ggircs@gov specifically for support reasons, so that one makes sense.